src/code/qualification/plotQualificationTimeProfile.m

```diff
@@ -83,7 +83,7 @@ function plotQualificationTimeProfile(WSettings,plotIndex,TimeProfile,ObservedDa
     curvesLegend=[curvesLegend legLabel];
     curvesHandle=[curvesHandle p_handle];
 end
-legend(curvesHandle, curvesLegend);
+addLegendWithLineBreaks(gcf,curvesHandle,curvesLegend);

```

Comment:
only for time profile? Other plot types do not have the "too long legend"-problem?

Reply:
@Yuri05 yes, I am not aware of issues elsewhere.